system/network/check-jsonwhois-domain-expiration.rb

```diff
@@ -0,0 +1,128 @@
+---
+type: CheckConfig
+api_version: core/v2
+metadata:
+  name: check-jsonwhois-domain-expiration
+  namespace: default
+spec:
+  command: check-jsonwhois-domain-expiration.rb -a {{ .labels.check_jsonwhois_domain_expiration_apikey }} -d foo.com,bar.com
```

Comment:
Would it make sense to consolidate this label to something a little more readable? Or would you prefer the name to be explicit?

Reply:
It is a little ridiculous but it's nice to have labels "namespaced" like this so that they don't conflict with any other labels for other plugins.